domain/modeldefaults/state/state.go

```diff
@@ -249,20 +249,18 @@ func (s *State) ModelMetadataDefaults(
 		return nil, errors.Trace(err)
 	}
 
-	stmt := `
-SELECT m.name, ct.type
+	modelUUID := modelUUID{UUID: uuid.String()}
+	var result modelMetadata
+	stmt, err := s.Prepare(`
```

Comment:
static analysis is crying because this error is not checked

Reply:
just pushed a fix!